End-of-Service Policy for Umbraco Cloud (#7729)
umbraco/UmbracoDocs: umbraco-cloud/explore-umbraco-cloud/what-is-umbraco-cloud/README.md

[baselines.md](../../begin-your-cloud-journey/creating-a-cloud-project/baselines.md)
{% endcontent-ref %}

## End-of-Service Policy for Umbraco Cloud

> The End-of-Service policy exists to clearly outline how long projects and software is support is supported and available on Umbraco Cloud. You can learn more about how long hosting is available for a given version of Umbraco CMS and more in the (End-of-Service policy for Umbraco Cloud)[end-of-service-policy.md]. 

Reviewer: An attempt to shorten these two sentences a bit.

```suggestion
The End-of-Service policy outlines how long projects and software are supported and available on Umbraco Cloud. Learn more about this policy for given versions of Umbraco CMS and more in the (End-of-Service policy for Umbraco Cloud)[end-of-service-policy.md]. 
```
